src/javascripts/ng-admin/Crud/list/ListController.js

```diff
@@ -46,10 +46,21 @@ define(function () {
         progression.start();
 
         this.ReadQueries
-            .getAll(this.view, page, true, this.search, this.sortField, this.sortDir)
-            .then(function (nextData) {
+            .getAll(this.view, page, this.search, this.sortField, this.sortDir)
+            .then(function (response) {
                 progression.done();
-                self.entries = self.entries.concat(nextData.entries);
+                var references = self.view.getReferences();
```

Comment:
Are you sure all of this is necessary. Because, on my project, the references were correctly retrieved, even the one that were not on the first page.
So it seems to me that the code in routing.js does the job, and don't need to be repeated.